omise/controllers/front/base.php

```diff
@@ -19,7 +19,7 @@
  * @see OmiseApiResource::genOptions() The function in library, Omise PHP, that uses this constant.
  */
 if (! defined('OMISE_API_VERSION')) {
-    define('OMISE_API_VERSION', '2015-11-17');
+    define('OMISE_API_VERSION', '2017-11-02');
```

Comment:
just wonder if there is any reason we don't go with the latest API version (2019)?

Reply:
No real reason - I copied it from another plugin where it was used and everything was known to be working